src/Service/SQS.service.js

```diff
@@ -33,6 +33,27 @@ const OFFLINE_MODES = {
   AWS: 'aws',
 };
 
+/**
+ * Defines the preferred behaviour
+ * for SQSService.prototype.publish
+ * should AWS SQS fail.
+ */
+export const SQS_PUBLISH_FAILURE_MODES = {
```

Comment:
Should this also be exported from lambda-wrapper's index?

Reply:
Fair point!

Reply:
60a7217d2cb5c5bd8ed0ce8568ec5cc02fe9fd0c

Reply:
I don't see a change to `index.js` in that commit... I'm not sure we need it though if the values are now defined in the parameter type?

Reply:
You are right! 7ca8ae931cf1ea1b779ac730d0b7215a4cba9ac2